atlasdb-conjure/build.gradle

```diff
@@ -4,7 +4,12 @@ dependencies {
   compile project(':atlasdb-commons')
   compile project(':atlasdb-remoting-api')
 
-  compile group: 'com.palantir.conjure.java.runtime', name: 'conjure-java-jaxrs-client'
+  implementation 'com.palantir.refreshable:refreshable'
+  implementation 'com.palantir.dialogue:dialogue-apache-hc4-client'
```

Comment:
I think dialogue-clients should give you everything you need - don't think you need `-serde` or `-hc4-client`.